articles/site-recovery/avs-tutorial-replication.md

```diff
@@ -168,7 +168,7 @@ Select and verify target resources.
 - The policy is automatically associated with the configuration server.
 - A matching policy is automatically created for failback by default. For example, if the replication policy is **rep-policy**, then the failback policy is **rep-policy-failback**. This policy isn't used until you initiate a failback from Azure.
 
-Note: In VMware-to-Azure scenario the crash-consistent snapshot is taken at 5 min interval.
+Note: In VMware vSphere-to-Azure scenario the crash-consistent snapshot is taken at 5 min interval.
```

Comment:
Please use "VMware-to-Azure"

Reply:
@Sharmistha-Rai - unfortunately VMware VMs is a misnomer and we need to follow the naming guidelines of VMware. VMware vSphere is VMware product that runs Virtual Machines.